types/backbone/index.d.ts

```diff
@@ -569,7 +569,7 @@ declare namespace Backbone {
         $el: JQuery;
         attributes: Record<string, any>;
         $(selector: string): JQuery;
-        render(): this;
+        render(...args: any[]): any;
```

Comment:
I strongly *disagree* with this change. Backbone has a clear convention for `View#render`: take no arguments, return `this`.

If you have a project that does this differently, that's fine, but please fix the type mismatch within your project. This change is making the type needlessly lenient for everyone else.

Reply:
On second thought, I agree. This goes against the point of strict typing. I'll get rid of this.